tests/monotouch-test/UIKit/UIListSeparatorConfigurationTest.cs

```diff
@@ -0,0 +1,36 @@
+//
+// UIListSeparatorConfiguration.cs
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+#if __WATCHOS__ || __IOS__
```

Comment:
I think you need to fix this condition too.

Reply:
```suggestion
#if __IOS__
```
Good catch